CONTRIBUTING.md

```
@@ -0,0 +1,32 @@
+
+
+## Contributing code
+We actively welcome your pull requests. You can find instructions on building the project in [README.md](https://github.com/canopas/group-track-android).
+1. Fork the repo and create your branch from `master`.  
+2. If you've added code that should be tested, add tests  
+4. Make sure your code lints.  
+
+## Labels
+Labels on issues are managed by contributors, you don't have to worry about them. Here's a list of what they mean:
+
+ * **bug**: feature that should work, but doesn't
```

Markdownlint (0.37.0)

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)